tc_driver/Cases.jl

```diff
@@ -211,6 +211,46 @@ update_forcing(::AbstractCaseType, grid, state, t::Real, param_set) = nothing
 initialize_forcing(::AbstractCaseType, forcing, grid::Grid, state, param_set) =
     initialize(forcing, grid, state)
 
+#####
+##### Pressure helper functions for making initial profiles hydrostatic.
+#####
+
+"""
+    Pressure derivative with height assuming:
+    - hydrostatic
+    - given θ_liq_ice and q_tot initial profiles
+"""
+function dp_dz!(p, params, z)
+    (; param_set, prof_θ_liq_ice, prof_q_tot) = params
+    thermo_params = TCP.thermodynamics_params(param_set)
+
+    θ_liq_ice = prof_θ_liq_ice(z)
+    q_tot = prof_q_tot(z)
+
+    q = TD.PhasePartition(q_tot)
```

Comment:
could we form a thermodynamic state given this assumption and use it below?

Reply:
I don't think so (?). We need to compute the pressure or density first. And then pass it to our thermo state constructor.